### DISCUSSION TITLE
Open multiple videos without merging

### DISCUSSION THREAD

**Discussion by ricardouk** [mifi/lossless-cut#834] [Ideas]
Hi everyone, i'm a first time user, english is not my native language so i apologize in advance if i cant explain what i mean properly.

I have a folder with around 50 videos, mostly 1080 @ 29.97 and 25 fps, is there a way to load all the files and do edits on them but without merging all of them, for example, at the moment the user loads a file and has a column on the right with a summary of all the edits, if i try to open several videos it merges them before the edit screen shows up, is there a way to display all the files in a column on the left and then after clicking on it we do the edits and save the videos, then click on the second video on the list and do the same and so on, this would avoid merging/loosing quality and to me it is more logical this way instead of having a superlong gigantic file to scrub.

Thanks for all the work you put on this project.

**Comment by mifi** [comment]
Hi. you mean like this? https://github.com/mifi/lossless-cut/issues/89

**Comment by ricardouk** [comment]
Hi mifi, yes, thats exactly what i had in mind, load files on the left and then do edits/export by going through the list, you suggested doing one by one but this way its simpler, no need to open/close dozens of files, just need to select file on the right, but its your program and there might be "coding dificulties" to get that done.

One other thing, say for example you have 12 videos and each one in 20 min long, we need to scrub through a 2 hour long timeline and "mouse precision" is "lower" its easy to skip through a short scene in a 2 hour long timeline, having them displayed on a list would make the timeline easier to navigate and do edits,  for example, i have 3 years worth of drone videos on a folder and the current way is "painfull" to do, it works and its a good program but probably intended for only a couple of videos.

Thanks for your work once again.

**Reply by mifi** [reply]
> Hi mifi, yes, thats exactly what i had in mind, load files on the left and then do edits/export by going through the list, you suggested doing one by one but this way its simpler, no need to open/close dozens of files, just need to select file on the right, but its your program and there might be "coding dificulties" to get that done.

So you mean being able to do the exact same cuts on 50 videos? e.g. you have 50 videos and you want to cut all of them from e.g. 01:02:03 to 01:03:00? If not I'm not sure what you mean, maybe you could visualize somehow.

**Comment by ricardouk** [comment]
Hi mifi, i´ll do my best to explain and give a "real life" example.

I have a folder with +100 drone videos from the region where i live, ive decided to do a quick video edit with all the historic monuments on those videos, firstly i tried avidemux but having to open...edit on keyframes...export each clip from the video, close the video and open the following one but that took ages specially if i had multiple "video extractions" from one single file, so i looked for another program and i found yours it allows to scru/seek the timeline and export multiple sections in "one go", but when i tried to open a couple of videos it merged them a created a super long timeline where one can easilly miss a scene, the longer the timeline the easier it was to find/miss small scenes. 

That led me to write here a suggestion, opening all the files on a list on the left side, upon clicking on a video on the list, the program would load that file, the user would do the edits and move on to the next on the list, this would be more simpler and intiutive (in my opinion) and avoids having those super long timelines and create merged files.

heres a quick mockup
https://i.imgur.com/zzMO2aN.jpeg

**Reply by mifi** [reply]
Thnx for your explanation. So it’s actually exactly the same as #89 that you’re looking for then. It’s a very frequently requested feature so I will probably prioritize it next

**Reply by ricardouk** [reply]
Thanks for "listening" mifi.

**Comment by ricardouk** [comment]
WOW, that was fast, thanks for the new version and added functions, the gui works as expected will try it in windows 10 and arch linux and will give some feedback, thanks for listening, will save loads of time, greetings from Portugal.

**Comment by ricardouk** [comment]
By the way, made a donation, cant give much but just wanted to say thank you.
